torch_xla/csrc/random.cpp

```diff
@@ -20,9 +20,7 @@ std::string GetDefaultGitGeneratorName() {
   XlaDeviceType hw_type =
       static_cast<XlaDeviceType>(bridge::GetCurrentDevice().type());
   switch (hw_type) {
-    case XlaDeviceType::GPU:
     case XlaDeviceType::CUDA:
```

Comment:
can't we throw a explicitly message when people set `PJRT_DEVUCE=GPU`, this is not backward compatible so I want to make sure user knows how to fix it.

Reply:
ok I saw below we say we want to remove that warning after 2.2.. fine haha